### PR TITLE
Hip 33 - Read a static property `ledger.id` from bootstrap properties

### PR DESCRIPTION
**Pull request by anighanta**
**Description**:

**Related issue(s)**:

Addresses #2738, [HIP-33](https://hips.hedera.com/hip/hip-33) feature.
All Entity getInfo responses will now have a ledgerId field which denotes which network this response is from.

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

**Review by tinker-michaelj — COMMENTED** [review]
As noted in the comment, I'm not sure `ledger.id` is really "special enough" to deserve a top-level static member in `ServicesState`. 😁 

I'd prefer to either expose it through the `NetworkInfo` as mentioned, or if that seems like too much, adding to [the `StaticPropertiesHolder`](https://github.com/hashgraph/hedera-services/blob/hts-contracts/hedera-node/src/main/java/com/hedera/services/context/properties/StaticPropertiesHolder.java).

File: hedera-node/src/main/java/com/hedera/services/context/primitives/StateView.java [hunk 1]
```diff
@@ -134,11 +138,13 @@ public StateView(
 			@Nullable TokenStore tokenStore,
 			@Nullable ScheduleStore scheduleStore,
```
Comment (Neeharika-Sompalli) [inline hedera-node/src/main/java/com/hedera/services/context/primitives/StateView.java:139]: nit : arguments can be `final`

File: hedera-node/src/main/java/com/hedera/services/context/primitives/StateView.java [hunk 2]
```diff
@@ -250,6 +257,35 @@ public StateView(
 		}
 	}
 
+	public Optional<ConsensusTopicInfo> infoForTopic(TopicID topicID) {
+
+		MerkleTopic merkleTopic = topics().get(EntityNum.fromTopicId(topicID));
```
Comment (Neeharika-Sompalli) [inline hedera-node/src/main/java/com/hedera/services/context/primitives/StateView.java:262]: nit : same here , can be final

**Review by Neeharika-Sompalli — APPROVED** [review]
LGTM

File: hedera-node/src/main/java/com/hedera/services/config/NetworkInfo.java [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+import com.google.protobuf.ByteString;
+import com.hedera.services.context.annotations.CompositeProps;
+import com.hedera.services.context.properties.PropertySource;
+
+import javax.inject.Inject;
+import javax.inject.Singleton;
+
+@Singleton
+public class NetworkInfo {
+	private static final String DEFAULT_LEDGER_ID = "0x03";
```
[the remark this answers is not in the document]
Comment (qnswirlds) [reply hedera-node/src/main/java/com/hedera/services/config/NetworkInfo.java:32]: Agree with @Neeharika-Sompalli, should add `// dev/preprod`

[the remark this answers is not in the document]
Comment (Neeharika-Sompalli) [reply hedera-node/src/main/java/com/hedera/services/config/NetworkInfo.java:41]: Added the comment

File: hedera-node/src/main/java/com/hedera/services/context/properties/BootstrapProperties.java [hunk 4]
```diff
@@ -370,6 +371,7 @@ public Object getProperty(final String name) {
 			entry("fees.tokenTransferUsageMultiplier", AS_INT),
 			entry("fees.percentCongestionMultipliers", AS_CONGESTION_MULTIPLIERS),
 			entry("files.maxSizeKb", AS_INT),
+			entry("ledger.id", AS_STRING),
```
Comment (tinker-michaelj) [inline hedera-node/src/main/java/com/hedera/services/context/properties/BootstrapProperties.java:374]: `AS_STRING` is [the default transform](https://github.com/hashgraph/hedera-services/blob/master/hedera-node/src/main/java/com/hedera/services/context/properties/BootstrapProperties.java#L300), so we don't need to repeat it here.

File: hedera-node/src/main/java/com/hedera/services/queries/consensus/GetTopicInfoAnswer.java [hunk 5]
```diff
@@ -107,40 +102,19 @@ public Response responseGiven(Query query, StateView view, ResponseCodeEnum vali
...
 			} else {
-				ConsensusTopicInfo.Builder info = infoBuilder(op, view);
-				response.setHeader(answerOnlyHeader(OK));
-				response.setTopicInfo(info);
+				var optionalInfo = view.infoForTopic(op.getTopicID());
+				if (optionalInfo.isPresent()){
+					response.setHeader(answerOnlyHeader(OK));
+					response.setTopicInfo(optionalInfo.get());
+				} else {
+					response.setHeader(answerOnlyHeader(FAIL_INVALID));
```
Comment (tinker-michaelj) [inline hedera-node/src/main/java/com/hedera/services/queries/consensus/GetTopicInfoAnswer.java:110]: I think `INVALID_TOPIC_ID` would make the most sense here; `FAIL_INVALID` should be reserved for an internal failure like an unhandled exception.

File: hedera-node/src/main/java/com/hedera/services/context/properties/BootstrapProperties.java (hunk 4 above)
Comment (Neeharika-Sompalli) [reply hedera-node/src/main/java/com/hedera/services/context/properties/BootstrapProperties.java]: removed the entry. Thanks !

File: hedera-node/src/main/java/com/hedera/services/queries/consensus/GetTopicInfoAnswer.java (hunk 5 above)
Comment (Neeharika-Sompalli) [reply hedera-node/src/main/java/com/hedera/services/queries/consensus/GetTopicInfoAnswer.java]: Agreed. Modified to `INVALID_TOPIC_ID`

**Review by tinker-michaelj — APPROVED** [review]
LGTM!

**Pull request merged**